packages/apps/mysql/values.yaml

```
@@ -55,11 +55,11 @@ backup:
   s3SecretKey: ju3eum4dekeich9ahM1te8waeGai0oog
   resticPassword: ChaXoveekoh6eigh4siesheeda2quai0
 
-## @param resources Resources
+## @param resources Explicit CPU and memory configuration for each MariaDB replica. When left empty, the preset defined in `resourcesPreset` is applied.
 resources: {}
  # resources:
  #   cpu: 4000m
  #   memory: 4Gi
  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)